docs/src/pages/components/data-grid/filtering/CustomFilterPanel.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { DataGridPro, GridLinkOperator } from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+export default function DisableSortingGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10,
+    maxColumns: 6,
+  });
+
+  return (
+    <div style={{ height: 400, width: '100%' }}>
+      <DataGridPro
+        {...data}
+        // components={{
```

Comment:
Remove commented-out code (or maybe it's still a WIP).

Reply:
In fact, I was looking for a way to show all the customization options in the same code, and comment on those which are not used in the demo. But yes, that's not explicit enough.